domains/sajidhussain.json

```diff
@@ -0,0 +1,10 @@
+{
+    "repo": "https://github.com/Sajid-tech/sajidhussain.github.io",
+    "owner": {
+        "username": "Sajid-tech",
+        "email": "s.khan9430319425@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://sajid-tech.github.io/sajidhussain.github.io"
```

Comment:
`CNAME` can't contain protocol.
`CNAME` can't contain `/`
(tip: rename your github website repo to `sajid-tech.github.io`)

Reply:
okay I have updated my JSON file as per your instruction, pls look into it